runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeItemPanel.java

```diff
@@ -104,7 +105,7 @@ public void mouseReleased(MouseEvent e)
 
 		// Item details panel
 		JPanel rightPanel = new JPanel(new GridLayout(3, 1));
-		rightPanel.setBackground(ColorScheme.MEDIUM_GRAY_COLOR);
+		rightPanel.setBackground(ColorScheme.TRANSPARENT);
```

Comment:
Are you sure you need to use transparent? Swing has usually bad performance with transparent components

Reply:
Transparent is unnecessary on right panel but is necessary on the alchAndLimit panel unless we want to attach a second mouse listener to that child panel.

Reply:
Transparent is unnecessary on right panel but is necessary on the alchAndLimit panel unless we want to attach a second mouse listener to that child panel.